docs/tutorial/index.md

```diff
@@ -1,37 +1,37 @@
 ---
-title: Gatsby.js Tutorials
+title: Gatsby.js tutoriaalit
 disableTableOfContents: true
 ---
 
-Welcome to Gatsby! We’re glad you’re here. The goal of this tutorial is to guide you through setting up and deploying your first Gatsby site using a starter template. As we walk through that process, we’ll introduce some more general web development topics, and go over the underlying structure of a Gatsby site.
+Tervetuloa Gatsbyyn! Olemme iloisia, että olet täällä. Tämän tutoriaalin tarkoituksena on ohjata sinut läpi valmisteluiden ja ensimmäisen Gatsby-sivustosi julkaisemisen aloitusmallia käyttäen. Prosessin aikana esittelemme sinulle joitakin yleisimpiä web-kehityksen aiheita ja käymme läpi, miten Gatsby-sivustot rakentuvat.
 
-> The full tutorial is intended to be as accessible as possible to people without much web development experience (yet!) — no need to be an expert. If you prefer to jump straight to code, feel free to skip the step-by-step tutorial and see the [quick start](/docs/quick-start/) page.
+> Tämän tutoriaalin tarkoitus on olla mahdollisimman helposti lähestyttävä ihmisille, joilla ei ole (vielä!) paljoa kokemusta web-kehittämisestä – sinun ei tarvitse olla ekspertti. Jos haluat mieluummin hypätä suoraan koodin pariin, voit jättää vaiheittaisen tutoriaalin väliin ja katsoa [pika-aloitussivun](/docs/quick-start).
 
-## Gatsby fundamentals
+## Gatsbyn perusteet
 
-0. [Set up your development environment](/tutorial/part-zero/): We'll introduce you to core technologies that power Gatsby, and guide you through setting up your development environment.
-1. [Get to know Gatsby building blocks](/tutorial/part-one/): Starting new projects, developing, and deploying sites.
-2. [Introduction to using CSS in Gatsby](/tutorial/part-two/): Explore libraries like Typography.js and CSS Modules.
-3. [Building nested layouts in Gatsby](/tutorial/part-three/): Layouts are sections of your site that are reused across multiple pages like headers and footers.
+0. [Pystytä kehitysympäristö](/tutorial/part-zero/): Esittelemme sinulle ydinteknologiat, joiden avulla Gatsby toimii, ja ohjaamme sinut kehitysympäristön pystyttämisen läpi.
+1. [Tutustu Gatsbyn rakennuspalikoihin](/tutorial/part-one/): Uusien projektien aloittaminen, kehitys, ja sivustojen julkaiseminen.
+1. [Johdanto CSS:n käyttöön Gatsbyssa](/tutorial/part-two/): Tutustu kirjastoihin kuten Typography.js ja CSS Modules.
```

Comment:
CSS Modules to CSS Moduulit.

Otherwise looks good to me!

Reply:
`CSS Modules` is a package and therefore should not be translated in my opinion.

Thanks for the numbering! My linter automatically corrects it to the 0-1-1 format. Does it not happen to you on the pre-commit hook?

Reply:
Ah okay. I didn't realize it was a package, my bad.

Doesn't happen to me.

Reply:
Okay 🤔 I need check what is causing that for me, it's quite annoying...